### PR TITLE
Created Python API for deploying model to Coral

### PR DESCRIPTION
**Pull request by SwapnilPande**
WIP #2 

Rewrote the deploy script to move the deploy functionality to a separate function. This allows the deploy script to be run from the command line OR from another python script (such as code for the UI).

### CODE REVIEW

**Comment by SwapnilPande** [comment]
[Create python interface for deploy script](https://trello.com/c/GMCRfzLV/17-create-python-interface-for-deploy-script)

File: coral_inference/deploy.py [hunk 1]
```diff
@@ -2,46 +2,49 @@
...
+    print("Transferring model to Anything Sensor...")
+    # SCPCLient takes a paramiko transport as an argument
+    with SCPClient(ssh.get_transport()) as scp:
+        scp.put(model, "/home/mendel/learn_ml/coral_inference/classification/classification_model.tflite")
+    print("Transfer Successful!")
+
+
+    # Start model execution
+    ssh.exec_command("pkill screen")
+    ssh.exec_command("cd /home/mendel/learn_ml/coral_inference/classification && screen -d -m python3 app.py")
```
Comment (alexbarnett12) [inline coral_inference/deploy.py:34]: app.py requires model passed in as the arg. Also, w/ the current structure of having a different MnistClassifier, app.py doesn't work for mnist inference. All that would be needed to fix this is to not rename the model to "classification_model". Later on once we have a standard Classifier and separate preprocessors we can go back to this. 

```suggestion
    ssh.exec_command("cd /home/mendel/learn_ml/coral_inference/classification && screen -d -m python3 app.py -m" + model)
```

File: coral_inference/deploy.py [hunk 2]
```diff
@@ -2,46 +2,49 @@
...
+        ssh.connect(hostname = address, username = "mendel", key_filename = identity_file)
+    else: # Connect using password
+        ssh.connect(hostname = address, username = "mendel", password = password)
+    print("Successfully connected to Anything Sensor v1!")
+
+    # Transfer model to coral
+    print("Transferring model to Anything Sensor...")
+    # SCPCLient takes a paramiko transport as an argument
+    with SCPClient(ssh.get_transport()) as scp:
+        scp.put(model, "/home/mendel/learn_ml/coral_inference/classification/classification_model.tflite")
```
Comment (alexbarnett12) [inline coral_inference/deploy.py:28]:
```suggestion
        scp.put(model, "/home/mendel/learn_ml/coral_inference/classification/" + model)
```

File: coral_inference/deploy.py [hunk 3]
```diff
@@ -2,46 +2,49 @@
 from scp import SCPClient
```
Comment (alexbarnett12) [inline coral_inference/deploy.py:2]:
```suggestion
''' Run this script to deploy a tflite model to the EdgeTPU. Currently, must be run within the coral_inference folder.
'''
from scp import SCPClient
```

File: coral_inference/deploy.py (hunk 2 above)
Comment (SwapnilPande) [reply coral_inference/deploy.py:28]: This looks good

File: coral_inference/deploy.py (hunk 1 above)
Comment (SwapnilPande) [reply coral_inference/deploy.py:34]: I don't know if you've tested this code specifically, but I THINK there needs to be a space after `-m` right?
So the line would be:
```
ssh.exec_command("cd /home/mendel/learn_ml/coral_inference/classification && screen -d -m python3 app.py -m " + model)
```

Comment (alexbarnett12) [reply coral_inference/deploy.py:34]: yep i just forgot that here. I've tested the code with that space on my coral and laptop

**Comment by SwapnilPande** [comment]
@alexbarnett12 Made the changes, review when you get the chance.

**Pull request merged**